python/tk_desktop/desktop_window.py

```diff
@@ -504,9 +503,11 @@ def show_update_project_config(self):
         self.update_project_config_widget.show()
         self.project_overlay.hide()
 
+    def _get_current_user(self):
+        return login.get_current_user(sgtk.platform.current_engine().tank)
```

Comment:
Is this inside the engine? Could you use `sgtk.platform.current_bundle()`? That's the standard way to get the app/engine object from code inside the app/engine (but only works if you have used `import_module`).